jenkins-preprod.sh

```diff
@@ -4,3 +4,4 @@ echo  $Password | gpg -o secure-data.tar --passphrase-fd 0 --batch /opt/keys/sec
 tar xvf secure-data.tar
 
 python scripts/generate_secrets.py secure-data/
+python scripts/generate_keys.py secure-data/
```

Comment:
I think we should call this script and remove our one
https://github.com/ONSdigital/sdc-cryptography/blob/master/sdc/crypto/scripts/generate_keys.py

Reply:
I think theres issues with the *kid_override* in the SR code, but we could submit a PR to the sdc-crypo repo

Reply:
@shahi645 this isn't an issue anymore as RRM have deployed a change so we no longer require these overrides